client/packages/common/src/intl/locales/en/common.json

```
@@ -1852,6 +1852,7 @@
   "messages.zero-line-quantities_many": "The quantity of {{count}} lines has been set to 0",
   "messages.zero-line-quantities_one": "The quantity of 1 line has been set to 0",
   "messages.zero-line-quantities_other": "The quantity of {{count}} lines has been set to 0",
+  "messages.unsaved-outbound-vvm-status": "VVM status will not be saved as there is no allocated quantity for this item! Click OK again to continue with unsaved changes.",
```

```suggestion
  "messages.unsaved-outbound-vvm-status": "VVM status will not be saved for lines with no allocated quantity. Click OK again to continue with unsaved changes.",
```

Nit... "item" is maybe a little unclear, if I change the status for 2 lines for an item, but only allocate one 👀